### PR TITLE
Added adjectives to the mountain.txt fil

### PR DESCRIPTION
**Pull request by ancasimon**
Added 26 adjectives to the mountain.txt file - please note that the Forest.txt file came through with changes pending - I did not make changes to that file.

### CODE REVIEW

File: name-segments/adjectives/mountain.txt
Comment (ianfhunter) [inline name-segments/adjectives/mountain.txt:41]:
```suggestion
```

**Review by ianfhunter — APPROVED** [review]

**Comment by ianfhunter** [comment]
Thanks Anca!

**Comment by ancasimon** [comment]
Thank you, @ianfhunter ! Would you please add the 'hacktoberfest-accepted' label to the PR? Unless you don't think it's appropriate - in which case it's fine - just thought I would ask! Thanks!

**Comment by ianfhunter** [comment]
It should be counted as I've labelled the entire repository. Please let me know if it does not work

**Comment by ancasimon** [comment]
> It should be counted as I've labelled the entire repository. Please let me know if it does not work

It's my first time doing Hacktoberfest, so you would know better 😄  but in the instructions, it says the PR will count if (1) it is in a project label Hacktoberfest AND (2) the PR is labeled as `hacktoberfest-accepted`:

"Your pull requests will count toward your participation if they are in a repository with the hacktoberfest topic and once they have been merged, approved by a maintainer or labelled as hacktoberfest-accepted."
From here: https://hacktoberfest.digitalocean.com/details

**Comment by ianfhunter** [comment]
> > It should be counted as I've labelled the entire repository. Please let me know if it does not work
> 
> It's my first time doing Hacktoberfest, so you would know better 😄 but in the instructions, it says the PR will count if (1) it is in a project label Hacktoberfest AND (2) the PR is labeled as `hacktoberfest-accepted`:
> 
> "Your pull requests will count toward your participation if they are in a repository with the hacktoberfest topic and once they have been merged, approved by a maintainer or labelled as hacktoberfest-accepted."
> From here: https://hacktoberfest.digitalocean.com/details

It says afterwards

OR
  Submitted in a repo with the hacktoberfest topic AND (
    The PR is merged OR
    The PR has been approved
  )
)
This should be the case here

**Pull request merged**
